client/components/product-icon/index.jsx

```diff
@@ -0,0 +1,112 @@
+/**
+ * External dependencies
+ */
+
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import {
+	isBloggerPlan,
+	isPersonalPlan,
+	isPremiumPlan,
+	isBusinessPlan,
+	isEcommercePlan,
+	getPlan,
+	getPlansKeys,
+	getPlanClass,
+} from 'lib/plans';
+import {
+	getProductClass,
+	getProductsKeys,
+	getProductShortName,
+	isJetpackBackupSlug,
+	isJetpackPlanSlug,
+} from 'lib/products-values';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export default class ProductIcon extends Component {
+	getPlanIcon( iconName ) {
+		const { product, className } = this.props;
+
+		const plan = getPlan( product );
+		const planTitle = plan.getTitle();
+		const planClass = getPlanClass( product );
+		const isJetpack = isJetpackPlanSlug( product );
+
+		return (
+			<img
+				src={ `/calypso/images/plans/${ isJetpack ? 'jetpack' : 'wpcom' }/plan-${ iconName }.svg` }
```

Comment:
I think this like would be a little more readable if we move the ternary in its own variable above the `return`.